packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -603,14 +602,18 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 valueWithEpoch.value,
             );
         };
+        const removeEntries = async () => {
```

Comment:
This is sufficient:
const removeEntries = async () => this.cache.persistedCache.removeEntries();